components/StyledInput.tsx

```diff
@@ -42,6 +42,7 @@ const StyledInput = styled.input`
   border-style: ${props => (props.bare ? 'none' : 'solid')};
   box-sizing: border-box;
   outline: none;
+  max-width: none;
```

Comment:
A bit concerned about the possible regressions that this may introduce, but not a blocker if it seems safe to you.

Reply:
There are a few places where the input increased a little bit, but I don't think anything looks broken.